setup.py

```diff
@@ -77,5 +77,8 @@ def get_version(package):
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
We don't cover 3.8 in test and we have dropped coverage for <= 3.5 so we should remove these